site/utils/apiUtils/index.ts

```
@@ -99,7 +100,7 @@ export const getPageState = <T>(errorCookie: string, schemaObject: z.ZodType<T>,
         return { ...(JSON.parse(errorCookie) as PageState<Partial<T>>), disruptionId: disruptionId || "" };
     }
 
-    if (disruptionId) {
+    if (disruptionId || !!data) {
```

should this be &&? what if there is no data do we want to throw an error?